.github/workflows/automatic-release.yml

```diff
@@ -55,3 +55,13 @@ jobs:
           "SIGNING_SECRET_KEY": ${{ secrets.SIGNING_SECRET_KEY }}
           "GIT_AUTHOR_NAME": ${{ secrets.GIT_AUTHOR_NAME }}
           "GIT_AUTHOR_EMAIL": ${{ secrets.GIT_AUTHOR_EMAIL }}
+
+      - name: "Create new milestones"
+        uses: "./"
+        with:
+          command-name: "laminas:automatic-releases:create-milestones"
+        env:
+          "GITHUB_TOKEN": ${{ secrets.ORGANIZATION_ADMIN_TOKEN }}
```

Comment:
I think the normal token suffices for this interaction

Reply:
Just checked, and @Ocramius is right here; there are no requirements outside of standard repo access (i.e., the owner of the token needs write access on the repo, but that's all).